index.js

```diff
@@ -3,13 +3,33 @@
 const { marked } = require('marked');
 const htmlLinkExtractor = require('html-link-extractor');
 
-module.exports = function markdownLinkExtractor(markdown, extended = false) {
+module.exports = function markdownLinkExtractor(markdown, checkAnchors = false) {
```

Comment:
Alternatively, we could pass in an Object. This way, we don't confuse people who have been used to `extended` - and we're future-proof to provide more configuration options.